### ISSUE TITLE
Create mobile app

### ISSUE THREAD

**Issue by rolies106** [gudeg-united/mishAPP#74]
Maybe we need create mobile app for this? who is capable for doing this?

**Comment by pakMann** [comment]
for sure, we will only 'sell' this app as mobile application, not web. but for now we will use phonegap / cordova.

**Comment by rolies106** [comment]
@blackbiron how about web version? should we deprecated it or?

**Comment by pakMann** [comment]
yup, we have to deprecate it

**Comment by rolies106** [comment]
At least we need to create landing page like path, just for portfolio

**Comment by pakMann** [comment]
you're right. indeed, i will make the mobile version like path. in the future we will add donation feature. user can track / view their donation archive on the web.

**Comment by rizarsyi** [comment]
agree
